internal/scheduler/interfaces/interfaces.go

```diff
@@ -0,0 +1,30 @@
+package interfaces
+
+import (
+	"github.com/armadaproject/armada/internal/armada/configuration"
+	"github.com/armadaproject/armada/internal/scheduler/schedulerobjects"
+)
+
+// LegacySchedulerJob is the job interface used throughout the scheduler.
+//
+// TODO: Rename to SchedulerJob.
```

Comment:
Do we need to rename this?  I was hoping that once we've deprecated the old lease.go, we can just move to a single concrete, job impl.

Reply:
Sure, I'll leave it.